### PR TITLE
Update to eslint 8, fix code accordingly

### PR DESCRIPTION
**Pull request by martinpitt**
These are from @KKoukiou , from your esbuild branch. I rebased that to current main, let's land these, they are useful. Thanks!

### CODE REVIEW

**Comment by martinpitt** [comment]
I figure we should update to ESLint 8 at the same time, your esbuild commit does that. Let's decouple this.

File: src/ContainerIntegration.jsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ const ContainerIntegration = ({ container, containerDetail, localImages }) => {
     const volumes = renderContainerVolumes(containerDetail.Mounts);
 
     const image = localImages.filter(img => img.Id === container.ImageID)[0];
-    const env = renderContainerEnv(containerDetail.Config.Env, image.Env || []);
+    const env = <ContainerEnv containerEnv={containerDetail.Config.Env} imageEnv={image.Env || []} />;
```
Comment (cockpituous) [inline src/ContainerIntegration.jsx:100]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx [hunk 2]
```diff
@@ -89,7 +89,7 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                   client.postPod(pod.isSystem, "stop", pod.Id, { t: 0 })
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to force stop pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```
Comment (cockpituous) [inline src/PodActions.jsx:92]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx [hunk 3]
```diff
@@ -100,7 +100,7 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                   client.postPod(pod.isSystem, "restart", pod.Id, {})
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to restart pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```
Comment (cockpituous) [inline src/PodActions.jsx:103]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx [hunk 4]
```diff
@@ -111,7 +111,7 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                   client.postPod(pod.isSystem, "restart", pod.Id, { t: 0 })
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to force restart pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```
Comment (cockpituous) [inline src/PodActions.jsx:114]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx [hunk 5]
```diff
@@ -126,7 +126,7 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                   client.postPod(pod.isSystem, "start", pod.Id, {})
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to start pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```
Comment (cockpituous) [inline src/PodActions.jsx:129]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx [hunk 6]
```diff
@@ -141,7 +141,7 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                   client.postPod(pod.isSystem, "unpause", pod.Id, {})
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to resume pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```
Comment (cockpituous) [inline src/PodActions.jsx:144]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx [hunk 7]
```diff
@@ -156,7 +156,7 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                   client.postPod(pod.isSystem, "pause", pod.Id, {})
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to pause pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```
Comment (cockpituous) [inline src/PodActions.jsx:159]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/ContainerIntegration.jsx (hunk 1 above)
Comment (cockpituous) [inline src/ContainerIntegration.jsx:100]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-062427-2b92d0d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx (hunk 2 above)
Comment (cockpituous) [inline src/PodActions.jsx:92]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-062427-2b92d0d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx (hunk 3 above)
Comment (cockpituous) [inline src/PodActions.jsx:103]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-062427-2b92d0d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx (hunk 4 above)
Comment (cockpituous) [inline src/PodActions.jsx:114]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-062427-2b92d0d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx (hunk 5 above)
Comment (cockpituous) [inline src/PodActions.jsx:129]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-062427-2b92d0d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx (hunk 6 above)
Comment (cockpituous) [inline src/PodActions.jsx:144]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-062427-2b92d0d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/PodActions.jsx (hunk 7 above)
Comment (cockpituous) [inline src/PodActions.jsx:159]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-062427-2b92d0d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
@KKoukiou : This is really from you, I approve :+1: Can you please formally approve this PR and land it?

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**